packages/botonic-react/src/webchat/replies.jsx

```diff
@@ -1,28 +1,35 @@
 import React, { useContext } from 'react'
 import { WebchatContext } from '../contexts'
+import styled from 'styled-components'
+
+const StyledReplies = styled.div`
+  overflow-x: auto;
+  padding-bottom: 10px;
+  margin-left: 5px;
+  margin-right: 5px;
+`
+
+const StyledRepliesContent = styled.div`
+  display: inline-block;
+  margin: 3px;
+`
 
 const alignOptions = /^left$|^center$|^right$/
 const wrapOptions = /^wrap$|^no-wrap$/
 
 export const WebchatReplies = props => {
   const { webchatState } = useContext(WebchatContext)
   return (
-    <div
+    <StyledReplies
       style={{
         ...(props.style || {}),
```

Comment:
We need to put always the default style options (textAlign and whiteSpace in this case) and then the `...(props.style || {})` in all the components, if not, the props will be overwrite and it can't be customized this values.
Check in all the components that this behaviour is not happening.